Framework/Utils/include/Utils/Utils.h

```diff
@@ -36,7 +37,7 @@ o2f::DataProcessorSpec defineBroadcaster(std::string devName, o2f::InputSpec usr
                                          size_t fixMsgSize);
 o2f::DataProcessorSpec defineBroadcaster(std::string devName, o2f::InputSpec usrInput, o2f::Outputs usrOutputs);
 
-using OutputBuffer = std::vector<char>;
+using OutputBuffer = std::vector<char,o2::memory_resource::polymorphic_allocator<o2::byte>>;
```

Comment:
I am worried about all this allocator business creeping to user code. How about we have an O2 specific smart pointer which hides all this? e.g.:

```
o2::ptr<std::vector<char>>
```

which internally holds a `std::vector<char,o2::memory_resource::polymorphic_allocator<o2::byte>>`